perceptia-one/perceptia/package.json

```diff
@@ -18,9 +18,12 @@
   "dependencies": {
     "@material-ui/core": "^3.9.3",
     "@material-ui/icons": "^3.0.2",
+    "bootstrap": "^4.3.1",
+    "mdbreact": "^4.15.0",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-app-polyfill": "^1.0.1",
+    "react-bootstrap": "^1.0.0-beta.8",
```

Comment:
Could you explain what you are using bootstrap for, given that we are already using the material-ui for styling.

Reply:
@nuo told me it's because he was having issues getting material-ui to center as he expected. I think you got it working in main scene, but i'm having a similar issue with the CircularProgress in the commit I just pushed to this branch. @spirrochet please take a look and see if you can fix it (i removed bootstrap, but you'll probably also want to remove the mdbreact package).